git-gh-pages: add page (#8787)
tldr-pages/tldr: pages/common/git-gh-pages.md

# git gh-pages

> Creates a new branch inside the current repository called `gh-pages`.

> > Part of git-extras.

Reviewer: ```suggestion
> Part of `git-extras`.
```

> > More information: https://github.com/tj/git-extras/blob/master/Commands.md#git-gh-pages

Reviewer: ```suggestion
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-gh-pages>.
```

> - Create Github pages (gh-pages) branch inside current local repository:

Reviewer 2: The branch name is in the command description section.
```suggestion
- Create the GitHub pages branch inside the the repository in the current directory:
```

Reviewer: Nice suggestion, I removed a `the` from your review comment.
```suggestion
- Create a GitHub pages branch inside the repository in the current directory:
```
I think this would be better, What do you think?

Reviewer 2: I'm not sure because it's a specific branch name dedicated to a specific function. `the` branch fits better than `a` random branch.

Reviewer: > I'm not sure because it's a specific branch name dedicated to a specific function. `the` branch fits better than `a` random branch.

Thanks, I think we can go with `the` then.

Author: You make a good point about current directory.
